dev/ProgressRing/ProgressRingAutomationPeer.h

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "ProgressRing.h"
+#include "ProgressRingAutomationPeer.g.h"
+
+class ProgressRingAutomationPeer :
+    public ReferenceTracker<ProgressRingAutomationPeer, winrt::implementation::ProgressRingAutomationPeerT>
+{
+
+public:
+    ProgressRingAutomationPeer(winrt::ProgressRing const& owner);
+
+    // IAutomationPeerOverrides
+    winrt::IInspectable GetPatternCore(winrt::PatternInterface const& patternInterface);
+    hstring GetClassNameCore();
+    winrt::hstring GetNameCore();
+    winrt::AutomationControlType GetAutomationControlTypeCore();
+
+    bool IsReadOnly() { return true; }
```

Comment:
I dont see this used anywhere.

Reply:
Maybe this is/was needed for the IValueRangeProvider to indicate that users can not change that progress?

Reply:
@chingucoding You are right! Turns out `IRangeValueProvider` was necessary to override `IsReadOnly()` to `true`. `IRanveValueProvider` has been re-added.